### ISSUE TITLE
Error: Unable to initialize this plugin

### ISSUE THREAD

**Issue by theEricPeters** [sll552/CorsairLED-MB-Plugin#12]
**Error:**
Unable to initialize this plugin:
Exception has been thrown by the target of an invocation.

**To Reproduce**
1. Extract the zip file, as directed, into the MusicBee plugins folder
2. Restart MusicBee if running
3. Open the options menu to the plugins section
4. Try to enable the plugin
5. Error occurs

**Screenshots**
![image](https://user-images.githubusercontent.com/19710360/55267315-d9219580-5257-11e9-80c3-97941794875b.png)

**Environment (please complete the following information):**
 - MusicBee Version: 3.2.6902
 - Plugin Version: 1.0.3
 - iCUE Version: 3.13.94

**Comment by sll552** [comment]
Thank for reporting, could you try if it still happens with the following build?

https://ci.appveyor.com/api/buildjobs/i7r5hv9rblo61fge/artifacts/CorsairLED-MB-Plugin-Release--49.zip

**Comment by theEricPeters** [comment]
Yeah, I deleted all of the plug-in files from the previous build, reinstalled with the build that you sent, and the error is still happening

**Comment by sll552** [comment]
Ok, as I cannot reproduce the issue, I was taking a guess. 

Could you provide the last entries of the error.log in MusicBee (Help -> Support -> View Error Log)

**Comment by theEricPeters** [comment]
I would, but all of the errors from the last few days are about my music library, not the plug-in?

```
3/29/19 7:41:02 PM - \\HOMEPC\Music\:Thread was being aborted
```

**Comment by sll552** [comment]
Well this was basically the last straw 😅 

Do you have the SDK enabled in your CUE Settings?

**Comment by theEricPeters** [comment]
yeah, I do

**Comment by sll552** [comment]
As I have currently no Idea what the problem could be, please try the following:

1. Do a complete uninstall of the plugin
  Remove the files in the plugin folder and in "%APPDATA%/MusicBee/CorsairLED"
  Start MusicBee and verify that the plugin is not loaded and no errors occur.
2. Install the plugin and try again

**Comment by theEricPeters** [comment]
I performed a complete uninstall, and reinstalled the plugin, but the error is still happening.

I'm not sure if this would really affect anything, but could it be because i'm running Windows 7?

**Comment by sll552** [comment]
Maybe, do you have .NET Framework 4+ installed ?

**Comment by theEricPeters** [comment]
Yeah, I'm fairly sure I do, but I can't verify that, since I'm not at my PC right now.

**Comment by sll552** [comment]
Is it possible for you to try it on a Win10 machine and your setup? 
I don't have a Win7 VM lying around and  you only get free VMs from Microsoft for x86 (https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) so I don't know if this would be an accurate test environment.

**Comment by theEricPeters** [comment]
that's not possible right now, but I probably could tomorrow afternoon, yeah.
